ddl/backfilling_operator.go

```diff
@@ -505,6 +534,24 @@ func (s *indexWriteResultSink) collectResult() error {
 	}
 }
 
+func (s *indexWriteResultSink) flush() error {
+	flushMode := ingest.FlushModeForceLocalAndCheckDiskQuota
+	if s.tbl.GetPartitionedTable() != nil {
+		flushMode = ingest.FlushModeForceGlobal
+	}
+	idxInfo := s.index.Meta()
+	_, _, err := s.backendCtx.Flush(idxInfo.ID, flushMode)
+	if err != nil {
+		if common.ErrFoundDuplicateKeys.Equal(err) {
+			err = convertToKeyExistsErr(err, idxInfo, s.tbl.Meta())
+		}
+		logutil.BgLogger().Error("flush error",
```

Comment:
The level shouldn't be error for duplicated key.

Reply:
Updated.